components/button/button.css

```diff
@@ -55,7 +55,7 @@
 .btn-float {
   position: absolute;
```

Comment:
as far as I think this should be position fixed. absolute will move out of viewport on scroll